src/app/debug/table/table.component.ts

```diff
@@ -172,13 +180,13 @@ export class TableComponent implements OnInit, OnDestroy {
     this.viewChange.next(this.viewSettings.currentViewName);
   }
 
-  listenForViewUpdate() {
+  listenForViewUpdate(): void {
     this.changeNodeLinkStrategyService.changeNodeLinkStrategy.subscribe(() => {
-      this.httpService.getViews().subscribe((views) => {
+      this.httpService.getViews().subscribe((views: Record<string, View>) => {
         this.viewSettings.views = views;
         this.sortFilterList();
-        let viewToUpdate = Object.keys(this.viewSettings.views).find(
-          (view) => view === this.viewSettings.currentView.name,
+        let viewToUpdate: string | undefined = Object.keys(this.viewSettings.views).find(
```

Comment:
Tiny change to make it look nicer
```suggestion
        let viewToUpdate?: string = Object.keys(this.viewSettings.views).find(
```